app/controllers/cards_controller.rb

```diff
@@ -2,4 +2,43 @@ class CardsController < ApplicationController
   def index
     @cards = Card.all
   end
+
+  def new
+    @card = Card.new
+  end
+
+  def create
+    @card = Card.new(card_params)
+    @card[:review_date] = 3.days.from_now
+    if @card.save
+      redirect_to :cards
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.